src/HaskellCI/YamlSyntax.hs

```diff
@@ -183,12 +186,12 @@ prettyYaml comment' = flatten . go where
     comment :: ann -> [String]
     comment = concatMap lines' . comment' where
         lines' "" = [""]
-        lines' s  = lines s 
+        lines' s  = lines s
 
     go :: Yaml ann -> NonEmpty (Int, Line)
     go (YString ann s) = case literal s of
         Just ss -> pure (0, Line (comment ann) ss)
-        Nothing -> pure (0, Line (comment ann) (shows s))
+        Nothing -> pure (0, Line (comment ann) $ escapeYAMLString s)
```

Comment:
There's another `escape` function (defined in `HaskellCI.Travis`) that is currently defined as:

```hs
escape :: Quotes -> String -> String
escape Double xs = show xs
```

Should this use `escapeYAMLString` as well?

Reply:
No, that one would need proper `escapeSHString`...